vector/src/main/java/im/vector/app/features/VectorFeatures.kt

```diff
@@ -26,7 +26,6 @@ interface VectorFeatures {
     fun isOnboardingUseCaseEnabled(): Boolean
     fun isOnboardingPersonalizeEnabled(): Boolean
     fun isOnboardingCombinedRegisterEnabled(): Boolean
-    fun isLiveLocationEnabled(): Boolean
```

Comment:
not for this PR, in the future I could imagine sharing the `VectorFeatures` with the lab features and having a single point of truth for toggling features